### DISCUSSION TITLE
NIST-DB 17 not appearing as identifier

### DISCUSSION THREAD

**Discussion by palsun123** [OpenChrom/openchrom#468] [Q&A] [answered]
Hi,

I followed the instructions on Wordpress (https://spectrometrylab.wordpress.com/2016/09/21/openchrom-and-nist-db/) on how to integrate NIST-DB 17 with OpenChrom 1.5.0, for the purpose of GC-flame ionization detection chromatograms. The issue is that I did not see an option within OpenChrom Preferences to select the nistms.exe app (which was indicated in the instructions), I only saw how to select the "MSSEARCH" library:

![image](https://github.com/OpenChrom/openchrom/assets/144840016/2cc7fe77-3197-42bc-bcba-3b390854285c)

I think the installation issue is the reason I am not able to find the NIST peak identifier when I have detected and selected my peaks:

![image](https://github.com/OpenChrom/openchrom/assets/144840016/b70c8c0d-502e-4c06-a1bd-3f3307c311d8)

What must I do to bring the NIST identifier into the OpenChrom software? Is it possible I'm missing a NIST file or app?

Thanks.

**Comment by eselmeister** [comment, marked as answer]
@palsun123 Yep, the "NIST (extern)" is an identifier only for GC-MS data. The menu also indicates, that this add-on is MSD specific "Identifier (MSD)". If you're searching for a NIST retention index based identification of peaks, please have a look at our "NIST (embedded)" add-on:

https://lablicate.com/marketplace/extensions/nist-integration-23-connector
